api/src/main/java/openconsensus/metrics/data/Value.java

```diff
@@ -86,6 +86,54 @@ public static Value summaryValue(Summary value) {
     return ValueSummary.create(value);
   }
 
+  /**
+   * Returns the double value of this {@link Value}.
+   *
+   * <p>This method should only be called with {@link Type#DOUBLE} {@link Value}.
+   *
+   * @return the double {@code Value}.
+   * @since 0.1.0
+   */
+  public double getDoubleValue() {
+    throw new UnsupportedOperationException("Not implemented.");
```

Comment:
"not implemented" makes wrong impression we want to implement it. Perhaps change the message to "This type can only return Foo data"

Reply:
String.format("This type can only return %s data", getType().name())??

Reply:
SGTM, updated.